scripting/include/get5.inc

```diff
@@ -174,6 +174,8 @@ native int Get5_IncreasePlayerStat(int client, const char[] statName, int amount
 #define STAT_TEAMKILLS "teamkills"
 #define STAT_SUICIDES "suicides"
 #define STAT_DAMAGE "damage"
+#define STAT_UTILITY_DAMAGE "util_damage"
+#define STAT_KNIFE_KILLS "knife_kills"
 #define STAT_HEADSHOT_KILLS "headshot_kills"
```

Comment:
@PhlexPlexico: As you can see, both `enemies_flashed` and `friendlies_flashed` are missing here.